### PR TITLE
Logos and icons.

### PR DESCRIPTION
**Pull request by YooSunYoung**
Static logo and icon images for documentation pages.

- [x] favicon.cio
- [x] logo.svg (text converted to path)
- [x] logo-dark.svg (text converted to path)
- [x] logo-text.svg (text as text with font indicated)

### CODE REVIEW

File: docs/_static/logo-text.svg
Comment (SimonHeybrock) [inline docs/_static/logo-text.svg:1]: Looks like wrong font on this one?

File: docs/_static/logo.svg
Comment (SimonHeybrock) [inline docs/_static/logo.svg:1]: Generally we keep the non-path version for enabling future modifications, typically in `resources/`. Why did you add it here? I think the docs use only two files, right?

**Review by SimonHeybrock — CHANGES_REQUESTED** [review]

File: docs/_static/logo.svg
Comment (YooSunYoung) [reply docs/_static/logo.svg:1]: Yes, sorry...! I will relocate this file.

File: docs/_static/logo-text.svg
Comment (YooSunYoung) [reply docs/_static/logo-text.svg:1]: I just copied scipp logo and made them into path though...?

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**